graphistry/umap_utils.py

```diff
@@ -286,8 +286,7 @@ def _bundle_embedding(self, emb, index):
         if emb.shape[1] == 2 and 'cudf.core.dataframe' not in str(getmodule(emb)):
             emb = pd.DataFrame(emb, columns=[config.X, config.Y], index=index)
         elif emb.shape[1] == 2 and 'cudf.core.dataframe' in str(getmodule(emb)):
-            import cudf
-            emb = cudf.DataFrame(emb, columns=[config.X, config.Y], index=index)
+            emb = pd.DataFrame(emb.to_numpy(), columns=[config.X, config.Y], index=index.to_numpy())
```

Comment:
We should stick w GPU vs CPU if we can

I thought we have an embedding in the form of a cudf df with multiple x y z etc cols to begin with, is it just we are being sloppy in how.we reference the cols ?